src/components/[hall]/GuildsByPlatform/GuildsByPlatform.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Tooltip,
+  useBreakpointValue,
+  useColorMode,
+} from "@chakra-ui/react"
+import { useWeb3React } from "@web3-react/core"
+import Card from "components/common/Card"
+import useIsServerMember from "components/[guild]/hooks/useIsServerMember"
+import useServerAccess from "components/[guild]/hooks/useServerAccess"
+import { PropsWithChildren } from "react"
+import { PlatformName } from "temporaryData/types"
+import Platform from "./components/Platform"
+
+type Props = {
+  platformType: PlatformName
+  platformName: string
+  guildIds: Array<number>
+}
+
+const GuildsByPlatform = ({
+  platformType,
```

Comment:
I wasn't sure if I should just pass the guilds array here as a prop, and handle the "single guild vs. multiple guilds" logic inside this component, and also get the `platformType` and `platformName` from the guilds array (`guilds[0].guildPlatforms[0].name` and `guilds[0].guildPlatforms[0].serverName`), or it's good this way?